requirements/prod.txt

```diff
@@ -1,8 +1,8 @@
-Django==1.11.29
+Django==2.0
 Pillow==4.3.0
 django-allauth==0.33.0
-django-braces==1.11.0
-django-cities-light==3.3.0
+django-braces==1.14.0
+django-cities-light==3.6.0
```

Comment:
We also should upgrade as many dependencies as we can, we don't want to ship packages that have vulnerabilities

Also https://pyup.io/safety/ is a great tool to check which softwares have safety issues

Reply:
Ok